docs/reference/custom resources/key-management-providers.md

```diff
@@ -4,13 +4,22 @@
 
 A `KeyManagementProvider` (`KMP`) represents key(s) and/or certificate(s) that are consumed by a verifier. `KMP` contains various providers for different use cases. Each provider is responsible for defining custom configuration and providing a set of public keys and/or x.509 certificates. Notation and Cosign verifiers can consume `KMP` resources to use during signature verification. Please refer to respective [Notation](../../plugins/verifier/notation.md) and [Cosign](../../plugins/verifier/cosign.md) verifier documentation on how to consume `KMP`.
 
+Key Management Provider can be defined as cluster-wide resources(using the kind `KeyManagementProvider`) or namespaced resources(using the kind `NamespacedKeyManagementProvider`).
+
+Namespaced Key Management Providers will only apply to the namespace in which they are defined. If a verification request targeting a namespace cannot find a KMP in required namespace, it will look up the cluster-wide KMPs.
+
+Cluster-wide KMPs are applied as the default global KMP if no namespaced KMP is specified in required namespace.
```

Comment:
can you clarify what this behavior means? Maybe with an example which will help.